package/src/custom/tests/School/unit-php/phpunit.xml.dist

```diff
@@ -0,0 +1,56 @@
+<!--
+/*
+ * Your installation or use of this SugarCRM file is subject to the applicable
+ * terms available at
+ * http://support.sugarcrm.com/Resources/Master_Subscription_Agreements/.
+ * If you do not agree to all of the applicable terms or do not have the
+ * authority to bind the entity as an authorized representative, then do not
+ * install or use this SugarCRM file.
+ *
+ * Copyright (C) SugarCRM Inc. All rights reserved.
+ */
+-->
+
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.1/phpunit.xsd"
+         backupGlobals="false"
+         backupStaticAttributes="false"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         processIsolation="false"
+         stopOnFailure="false"
+         bootstrap="../../../../tests/unit-php/TestBootstrap.php"
+         beStrictAboutOutputDuringTests="true"
+         beStrictAboutTestsThatDoNotTestAnything="true"
+         beStrictAboutTestSize="true"
+         beStrictAboutCoversAnnotation="false"
+>
+
+    <php>
+        <ini name="intl.default_locale" value="en"/>
+        <ini name="intl.error_level" value="0"/>
+        <ini name="memory_limit" value="-1"/>
+    </php>
+    <testsuites>
+        <testsuite name="all">
+            <directory>./</directory>
+            <directory>modules/</directory>
+        </testsuite>
+    </testsuites>
+    <filter>
+        <whitelist processUncoveredFilesFromWhitelist="false">
+            <directory suffix=".php">../../data/</directory>
```

Comment:
None of these directories are from your package. You only care about code coverage for the code you write, so this should probably be a list of directories or files that are installed by your package. If you tell me the paths of all of the files that are installed, then I can help you figure out what entries need to be here.

https://phpunit.de/manual/current/en/appendixes.configuration.html#appendixes.configuration.whitelisting-files

Reply:
Great catch!  I've set it (as a first start) to just get code coverage of the custom directory.  It's tricky as the custom directory gets some files when Sugar is installed that are not related to our Professor M MLP.  I'm concerned that if we get more specific with the file listings, we'll forget to update this as we add more customizations.

Here is a list of all of the files I sync from my installed Sugar directory to my git repo.  I'm thinking we don't want a list like this in phpunit.xml.dist.  What do you think?
          - /Applications/MAMP/htdocs/profm711unittestsb/tests/unit-php/custom/**/*
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/gradebook_fake/**/*
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/tests/**/*
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Schedulers/**/*
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Contacts/Ext/LogicHooks/profm.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/modules/Accounts/Accounts_Save.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/modules/Accounts/logic_hooks.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/modules/Contacts/Students_Gradebook.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/**/*professor*.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/**/*Professor*.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Accounts/Ext/Vardefs/**/*
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Accounts/Ext/clients/base/layouts/subpanels/_overridesubpanel-for-accounts-opportunities.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Calls/Ext/clients/base/layouts/subpanels/_overridesubpanel-for-calls-opportunities.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Calls/Ext/Vardefs/rli_link_workflow.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Campaigns/Ext/Layoutdefs/_overrideCampaign_subpanel_opportunities.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Contacts/Ext/clients/base/layouts/subpanels/_overridesubpanel-for-contacts-opportunities.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Contacts/Ext/Vardefs/**/*
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Documents/Ext/Layoutdefs/_overrideDocument_subpanel_opportunities.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Documents/Ext/Vardefs/rli_link_workflow.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Leads/Ext/Vardefs/**/*
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Meetings/Ext/Vardefs/**/*
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Notes/Ext/clients/base/layouts/subpanels/_overridesubpanel-for-notes-opportunities.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Notes/Ext/Vardefs/**/*
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Opportunities/Ext/Dependencies/opp_disable_dep.ext.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Opportunities/Ext/Vardefs/**/*
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Products/Ext/clients/base/layouts/subpanels/_overridesubpanel-for-products-opportunities.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Products/Ext/Vardefs/**/*
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Project/Ext/Layoutdefs/_overrideProject_subpanel_opportunities.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Project/Ext/Vardefs/**/*
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Quotes/Ext/clients/base/layouts/subpanels/_overridesubpanel-for-quotes-opportunities.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Quotes/Ext/Vardefs/**/*
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/RevenueLineItems/Ext/clients/base/layouts/subpanels/_overridesubpanel-for-revenuelineitems-opportunities.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/RevenueLineItems/Ext/Vardefs/**/*
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Tasks/Ext/clients/base/layouts/subpanels/_overridesubpanel-for-tasks-opportunities.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/Extension/modules/Tasks/Ext/Vardefs/**/*
          - /Applications/MAMP/htdocs/profm711unittestsb/icons/**/*Professor*.*
          - /Applications/MAMP/htdocs/profm711unittestsb/language/application/en_us.lang.php
          - /Applications/MAMP/htdocs/profm711unittestsb/metadata/**/*professor*.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/modules/Accounts/clients/base/views/list/list.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/modules/Accounts/clients/base/views/record/record.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/modules/Contacts/clients/base/views/list/list.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/modules/Contacts/clients/base/views/record/record.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/modules/Leads/clients/base/filters/default/default.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/modules/Leads/clients/base/layouts/convert-main/convert-main.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/modules/Leads/clients/base/views/list/list.php
          - /Applications/MAMP/htdocs/profm711unittestsb/custom/modules/Leads/clients/base/views/record/record.php
          - /Applications/MAMP/htdocs/profm711unittestsb/modules/PR_Professors/**/*
          - /Applications/MAMP/htdocs/profm711unittestsb/modules/RevenueLineItems/metadata/studio.php